cli/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "typeshare-cli"
-version = "1.10.0-beta.7"
+version = "1.10.0-beta.8"
```

Comment:
Similar to the changelog, the version bumps are part of the release process as well, so we should leave these out of specific feature PRs.

Reply:
reverted